libs/sdk-ui-dashboard/src/model/commandHandlers/filterContext/dateFilter/removeDateFiltersHandler.ts

```diff
@@ -53,8 +53,11 @@ export function* removeDateFiltersHandler(ctx: DashboardContext, cmd: RemoveDate
 
         invariant(!isUriRef(localIdentifier));
         const affectedDependentFilters = allAttributeFilters.filter((item) => {
-            return item.attributeFilter.filterElementsByDate?.some((parent) => {
-                return localIdentifier?.identifier === parent.filterLocalIdentifier;
+            return item.attributeFilter.filterElementsByDate?.some((depdendentDateFilter) => {
+                return (
+                    localIdentifier?.identifier === depdendentDateFilter.filterLocalIdentifier &&
+                    depdendentDateFilter.isCommonDate === false
```

Comment:
Is this the correct condition? What if it is undefined? Isn't in then specific date as well? Possibly 
 `!depdendentDateFilter.isCommonDate` would be better?

Reply:
Updated it with `!depdendentDateFilter.isCommonDate`